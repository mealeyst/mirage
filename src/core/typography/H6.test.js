import React from 'react'
import 'jest-styled-components'

import { H6 } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) H6', () => {
  const createH6 = (props) => {
    return shallowWithTheme(<H6 {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createH6())
    .toMatchSnapshot()
  })
})
