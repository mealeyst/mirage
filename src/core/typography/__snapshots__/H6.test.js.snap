// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) H6 matching the snapshot 1`] = `
.c0 {
  -webkit-letter-spacing: .075rem;
  -moz-letter-spacing: .075rem;
  -ms-letter-spacing: .075rem;
  letter-spacing: .075rem;
  color: #00003C;
  text-align: inherit;
  font-family: "ff-din-web",Helvetica,Arial,sans-serif;
  font-size: 1.4rem;
  font-weight: 500;
  font-style: italic;
  line-height: 1;
  margin: 3.2rem 0;
}

<h6
  className="c0"
  fontSize="1.4rem"
/>
`;
